### PR TITLE
feat: add fastAPI backend generator

### PR DESCRIPTION
**Pull request by EMNico**

### CODE REVIEW

File: generators/fast-api/templates/docker-compose.yaml.ejs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+services:
+    <%= packageName %>:
+        build:
+            context: <%= packagePath %>/
+            volumes:
+            - ./src:/opt/pysetup
+        environment:
+            DATABASE_URL: "postgresql+psycopg2://user:password@postgres:5432"
```
Comment (jvasseur) [inline generators/fast-api/templates/docker-compose.yaml.ejs:8]: This should be contains the database name (which is the same as the package name by default).
```suggestion
            DATABASE_URL: "postgresql+psycopg2://user:password@postgres:5432/<%= packageName %>"
```

File: package.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
     "ramda": "^0.27.1",
     "tslib": "^2.3.1",
     "yaml": "^1.10.2",
-    "yeoman-generator": "^5.5.2"
+    "yeoman-generator": "^5.5.2",
+    "yo": "^4.3.0"
```
Comment (jvasseur) [inline package.json:19]: This shouldn't be in the dependencies of the project.

**Review by jvasseur — CHANGES_REQUESTED** [review]
A few comments for the generator architecture part but I didn't review fully the python part.

File: package.json (hunk 2 above)
Comment (EMNico) [reply package.json:19]: indeed x(

**Comment by EMNico** [comment]
(BTW, this is still a WiP since it's missing the ansible part, which i'll complete when we will implement it on our project)

**Review by jvasseur — APPROVED** [review]
I think we can merge it like this and add an issue for the ansible configuration.

**Pull request merged**